uaclient/cli/__init__.py

```diff
@@ -69,9 +70,18 @@
 from uaclient.timer.update_messaging import refresh_motd, update_motd_messages
 from uaclient.yaml import safe_dump, safe_load
 
-NAME = "pro"
+EPILOG_TMPL = (
```

Comment:
I think this should somehow end up in the `messages` module, as it can be translated. IMHO keeping it here will make the i18n effort harder I guess

Reply:
same for other visible messages in this module